### PR TITLE
Use QUOTE_ATOM instead of KEYWORD_TO_ATOM["q"]

### PR DESCRIPTION
**Pull request by aqk**
See https://github.com/Chia-Network/clvm/pull/50

### CODE REVIEW

**Comment by aqk** [comment]
Note that the tests will fail without https://github.com/Chia-Network/clvm/pull/50

**Comment by arvidn** [comment]
would it be reasonable to land the `clvm` changes first and retrigger the CI here to see it turn green?

File: stages/stage_0.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 def run_program(
     program,
     args,
-    quote_kw=KEYWORD_TO_ATOM["q"],
+    quote_kw=QUOTE_ATOM,
     apply_kw=KEYWORD_TO_ATOM["a"],
```
Comment (arvidn) [inline stages/stage_0.py:13]: it still bothers be that the names of these parameters are "keywords" when they are "atoms"

**Review by arvidn — APPROVED** [review]

File: stages/stage_0.py (hunk 1 above)
Comment (aqk) [reply stages/stage_0.py:13]: Me too - I'll get to stuff like that after landing the quote change.

**Comment by aqk** [comment]
> would it be reasonable to land the clvm changes first and retrigger the CI here to see it turn green?

Definitely. I also want to resolve the mutual clvm / clvm_tools dependency after my current task.

**Comment by aqk** [comment]
> would it be reasonable to land the clvm changes first and retrigger the CI here to see it turn green?

Hmm, actually, because of the mutual dependency, this is not reasonable.

**Pull request merged**